csss-site/src/events/templates/events/fall_hacks2020.html

```diff
@@ -0,0 +1,26 @@
+{% extends "csss/new_header.html" %}
+
+{% block content %}
+{{ block.super }}
+    <title>Fall Hacks 2020</title>
+
+<style>

```

Comment:
please put these in a separate css file. I believe that is best practice.

Reply:
you hit resolve but did not change it?

Reply:
Correct, I think it's more convent in this file. I did remove the title.